skglm/solvers/prox_newton.py

```diff
@@ -24,6 +24,9 @@ class ProxNewton(BaseSolver):
     tol : float, default 1e-4
         Tolerance for convergence.
 
+    fit_intercept : bool, default True
```

Comment:
My experience with sklearn is that it's quite confusing to fit an intercept by default. Should it be set to False here ?

Reply:
agree!